src/css/layout/ScrollbarLayout.css

```diff
@@ -14,24 +14,16 @@
   outline: none;
   overflow: hidden;
   position: absolute;
-  transition-duration: 250ms;
```

Comment:
Scrollbar track used to widen on hover, needed that to remain constant for this fix. Now only the scrollbar face (handle) widens.